src/schema/v2/fair.ts

```diff
@@ -199,6 +206,25 @@ export const FairType = new GraphQLObjectType<any, ResolverContext>({
         // field in Gravity and then update it here.
         resolve: ({ kaws_collection_slugs }) => kaws_collection_slugs,
       },
+      ...(useUnstitchedMarketingCollections && {
```

Comment:
To avoid having many `useUnstitchedMarketingCollections && conditions`, maybe I should merge the unstitched/stitched SDL based on the flag? @mzikherman

Reply:
I was expecting to see `useUnstitchedMarketingCollections && ` type conditions all the places where we were stitching marketing collections (and now no longer want to), and all the places within the MP schema that replace that stitched version.

I'm not particularly worried or think its _bad_ it occurs in quite a lot of places - that's all the places touched by the migration! To me it's just the necessary work to perform the migration.

Definitely open to alternative setups!